tasks/task/task.py

```diff
@@ -23,7 +23,7 @@ def run(self):
                     self.device.click(CLAIM)
                     logger.info("Click Claim")
                     continue
-                if self.match_color(CLAIMED) and self.match_color(CLAIMED_ALL):
+                if not self.match_color(CLAIMED) and not self.match_color(CLAIMED_ALL):
```

Comment:
The original one should be right? `CLAIMED` is the gray button.

Reply:
Sorry, I got confused with the CLAIM and CLAIM_ALL buttons.